src/Sites/twitch.tv/Runtime/ChatListener.ts

```diff
@@ -86,7 +86,7 @@ export class TwitchChatListener {
 			const modMsg = msg as unknown as Twitch.ChatMessage.ModerationMessage;
 
 			if (modMsg.moderationType === 1) { // Timeout
-				this.sendSystemMessage(`${modMsg.userLogin} was timed out for ${humanizeDuration(modMsg.duration * 1_000)}${!!modMsg.reason ? ` (${modMsg.reason})` : ''} by a moderator`);
+				this.sendSystemMessage(`${modMsg.userLogin} was timed out for ${formatDuration(intervalToDuration({start: 0, end: modMsg.duration * 1_000}))}${!!modMsg.reason ? ` (${modMsg.reason})` : ''} by a moderator`);
```

Comment:
Is there a reason for the underscore at `modMsg.duration * 1_000`?
Also, could you call the `intervalDuration `as a const above rather than inline?

Reply:
- The reason for underscore is to determine a thousand parts. For example it's more readable when you see 1_000_000 instead of 100000. I can change it to be 1000 if you wish.
- Sure I'll move it to const

Reply:
@AnatoleAM done